source/_integrations/scene.markdown

```diff
@@ -141,6 +141,8 @@ Whenever you make a change to your scene configuration, you can call the `scene.
 
 ## Creating scenes on the fly
 
+<lite-youtube videoid="JW9PC6ptXcM" videotitle="Scenes on Steroids in Home Assistant - How To - Tutorial" posterquality="maxresdefault"></lite-youtube>
```

Comment:
I'm not fully against additions like these. Videos can help a lot.

However, I do think it should not be the first thing in the paragraph. As a matter of fact, it might be better to have it at the end of the paragraph, introducing the video as well.

At that moment the video has context to what the user just read.

../Frenck